packages/components/psammead-bulletin/src/__snapshots__/index.test.jsx.snap

```diff
@@ -279,10 +279,12 @@ exports[`Bulletin should render audio correctly 1`] = `
         dir="ltr"
       >
         <a
+          aria-labelledby="bulletin-https://bbc.co.uk"
```

Comment:
I guess this is just an example? Having a url in an id looks rather strange...

Reply:
Yes, I hope there is a better id in Simorgh